### PR TITLE
Add a disclaimer to communicate Slow Zone changes

### PR DESCRIPTION
**Pull request by PatrickCleary**
Temporary disclaimer about slow zone calculation changes.

IMG_4B47DA124538-1 3

### CODE REVIEW

File: src/slowzones/SlowZones.tsx [hunk 1]
```diff
@@ -151,6 +156,29 @@ export const SlowZones = () => {
...
+          userSelect: 'none',
+          backgroundColor: '#585858',
+          minHeight: '34px',
+        }}
+      >
+        <p style={{ display: 'inline', fontSize: '1.6rem' }}>ℹ️</p>
+        <p style={{ display: 'inline', color: 'white', textAlign: 'center', fontSize: '.9rem' }}>
+          <i>Our slow zone algorithm changed. </i>
+          <a href="https://transitmatters.org/blog/slowzonesupdate">
+            <u style={{ color: '#a2c6ff' }}>Learn&nbsp;More</u>
```
Comment (mathcolo) [inline src/slowzones/SlowZones.tsx:178]: More should not be capitalized

**Review by mathcolo — COMMENTED** [review]
Screenshot, would you mind?

**Review by mathcolo — APPROVED** [review]
This is good for when tomorrow comes

**Review by austinjpaul — APPROVED** [review]
LGTM! Will have to sync up to main before going in

**Pull request merged**